src/parser/core/modules/AlwaysBeCasting.js

```diff
@@ -35,7 +35,7 @@ export default class AlwaysBeCasting extends Module {
 			requirements: [
 				new Requirement({
 					name: <Trans id="core.always-cast.gcd-uptime">GCD Uptime</Trans>,
-					percent: this.gcd.getUptime() / fightDuration * 100,
+					percent: Math.min(this.gcd.getUptime() / fightDuration * 100, 100),
```

Comment:
I'd rather this be handled in a display-only manner on the requirement itself.

While it won't have much effect here, in a scenario with multiple requirements, capping one will cause the overall % to skew.